packages/core/src/RenderPipeline/RenderQueue.ts

```diff
@@ -20,6 +26,50 @@ export class RenderQueue {
     return a.priority - b.priority || b.distanceForSort - a.distanceForSort;
   }
 
+  private static _getCustomStencilStates(
```

Comment:
It is recommended to split, render Mask and use Mask are different behaviors, and merging them is risky

Reply:
done